ClientTests/PanelDataObserversTests.swift

```diff
@@ -36,4 +36,26 @@ class PanelDataObserversTests: XCTestCase {
 
         waitForCondition(timeout: 5) { delegate.didInvalidateCount == 3 &&  delegate.willInvalidateCount == 3 }
     }
+    
+    func testHighlightsCacheInvalidation() {
+        let profile = MockProfile()
+        let observer = ActivityStreamDataObserver(profile: profile)
+        let delegate = MockDataObserverDelegate()
+        observer.delegate = delegate
+        
+        // Set to 20min since refresh
+        profile.prefs.setLong(OneMinuteInMilliseconds * 20, forKey: PrefsKeys.ASLastInvalidation)
```

Comment:
This is setting the last refresh to simply be Jan1,1970 + 20mins if I'm not mistaken. The test still works but I think this might throw someone off if they read the test